homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used in developing websites
```

Comment:
Hi Aaishah,

The answer is correct. Can your expand on your answer? for example what features does JavaScript have as a programming language? What effect does JavaScript have on a website when it is used?

Reply:
Thanks Ironkamosi, I'll look into that